partner/partner-management-service/src/main/java/io/mosip/pms/partner/controller/PartnerServiceController.java

```diff
@@ -7,6 +7,7 @@
 import javax.validation.constraints.NotNull;
 
 import io.mosip.pms.partner.dto.CertificateDto;
+import io.mosip.pms.partner.response.dto.*;
```

Comment:
remove this import, import each file separately

Reply:
imported each file in separate